hack/release.go

```diff
@@ -136,6 +138,10 @@ func main() {
 	if *retrieveSha {
 		resolveImagesSha(&release)
 	}
+	parts := strings.Split(release.NetworkOperator.Version, ".")
+	release.OcpDefaulChannel = fmt.Sprintf("%s.%s", parts[0], parts[1])
```

Comment:
Do we want to generate OCP channels list (e.g. `stable,v24.4.1`) too?

Reply:
We use this one for OpenShift installation via CLI where we need a single channel, not a list:
![image](https://github.com/user-attachments/assets/a15ea994-97c3-4cb0-bfd1-aaf73cccac38)

Reply:
cant we always specify default here ? then install specific operator version ?